### ISSUE TITLE
request: "Create Update Delete" channel commands

### ISSUE THREAD

**Issue by favna** [skyra-project/skyra#1933]
**Is your feature request related to a problem? Please describe.**

Currently it is not possible to create, read, update or delete a channel through Skyra commands. This is something that can be added. 

**Describe the solution you'd like**

s!channel <create|read|update|delete> <args>

Create takes args:
- name (required)
- type (required)
- roles (rest, optional, if provided this becomes a private channel)

Read takes args:
- channelName (required)

Update takes args:
- channelName (required)
- name (required)
- type (required)
- roles (rest, optional, if provided this becomes a private channel)

Delete takes args:
- channelName (required)

Not sure if we want to handle permission overwrites here because that'll be a lot of maintenance work with all the features Discord is adding. Not to mention that it'll be hard to parse I think. 

**Describe alternatives you've considered**

Creating through discord UI

**Additional context**

Requested by YashuGamer cg here: https://discord.com/channels/254360814063058944/382531782387695616/862678116148445205

**Comment by YashuGamer0001** [comment]
Thanks favna
